src/main/java/com/echobox/api/linkedin/client/Parameter.java

```diff
@@ -85,7 +85,7 @@ private Parameter(String name, Object value, JsonMapper jsonMapper) {
    * <p>
    * The {@code value} parameter is often a {@link String} or primitive type like {@link Integer}, 
    * but you may pass in a {@link java.util.List}, {@link java.util.Map}, or your own 
-   * <tt>@LinkedIn</tt>-annotated Javabean and it will be converted to JSON automatically. 
+   * {@code LinkedIn-annotated} Javabean and it will be converted to JSON automatically.
    * 
```

Comment:
{@code LinkedIn-annotated} or {@code LinkedIn}-annotated

Reply:
How about `{@link LinkedIn} annotated`?

Reply:
Sure, it seems that is how the original one should have looked like.